static_website/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "account" {
+}
+
+variable "region" {
+}
+
+variable "environment" {
+}
+
+variable "nubis_domain" {
+  default = "nubis.allizom.org"
+}
+
+variable "technical_owner" {
+  default = "infra-aws@mozilla.com"
+}
+
```

Comment:
In response to adding a UUID to the bucket, I could use the UUID that is generated via the stack via the deploy however since this is an external module I don't have it exposed anyway. The only way I found that I can access this information is by doing this:

```
data aws_cloudformation_stack "uuid" {
  name  = "uuid-ci-admin"
}
```

And then I reference it this way

```
resource aws_s3_bucket "bucket" {
  bucket = "bucket-${data.aws_cloudformation_stack.uuid.outputs["UUID"]}"
}
```

This method I feel isn't really ideal since we need to actually assume that the stack uuid-ci-admin is available and doesn't change